### ISSUE TITLE
Why did there is a non-FPS limit ?

### ISSUE THREAD

**Issue by LaReunion974** [wheremyfoodat/Panda3DS#188]
Games run so fast like every game i have tested.

**Comment by wheremyfoodat** [comment]
Hello
Yeah you're right, this should be implemented eventually. I wanted to do mixed sync like Redream at first (syncing to audio *and* video) but I will likely have to implement a temporary solution until audio is implemented.

**Comment by NintendoManiac64** [comment]
On a related note, I want to at least put out a sort of "ideal end goal" request that it be similar to the way Dolphin, mGBA, and Duckstation handle their FPS cap limiter where you can not only just specify what sort of FPS cap you want but it'll even sync the audio pitch accordingly so that you don't get weird and/or poor-quality sounding audio, especially when running at slower-than-native speeds.

In other words, if you cap a 60fps game to 30fps, it'd run both the game and audio at half-speed but also have the audio play back at half the sample rate and therefore subsequently half the pitch (e.g. a 1000Hz sine wave with a sample rate of 48kHz played back at half speed is identical to a bog-standard 500Hz sine wave with a sample rate of 24kHz).
